test/e2e-tests.sh

```diff
@@ -43,13 +43,13 @@ failed=false
 
 go_test_e2e -timeout=30m ./test/e2e_new/... || failed=true
 
-go_test_e2e -timeout=30m ./test/e2e/... || failed=true
+#go_test_e2e -timeout=30m ./test/e2e/... || failed=true
```

Comment:
If you remove the comments the test fails?

Reply:
If that's ok for you, can we disable the flaky test or revert this patch https://github.com/knative-sandbox/eventing-kafka-broker/commit/63c8083b249a4be5f342a27555482bbe6fd8f7df and then tackle it after the release? I can take care of fixing the issue.

Reply:
I've just checked again the fix and it seems correct to me, so maybe it's just the test?

Reply:
The test is not in these suites, it's in `e2e_new`, I disabled those for getting a failure.

Reply:
Can you do it in this PR? Otherwise i'll do it in another one